Duckling/Time/CA/Rules.hs

```diff
@@ -125,7 +125,7 @@ ruleEvening = Rule
     [ regex "vespre"
     ]
   , prod = \_ ->
-      let from = hour False 19
+      let from = hour False 18
```

Comment:
why did this change?

Reply:
"vespre" means evening. Being CA a language that has both evening and night, I think 18-22 represents a better time range for it.